tests/errors_test.py

```diff
@@ -380,10 +380,12 @@ def test_grad_norm(self):
 
 
 class CustomErrorsTest(jtu.JaxTestCase):
-  @parameterized.named_parameters(jtu.cases_from_list(
-    {"testcase_name": f"_{errorclass}", "errorclass": errorclass}
+  @jtu.sample_product(
+    [dict(errorclass=errorclass)
      for errorclass in dir(jax.errors)
-     if errorclass.endswith('Error') and errorclass not in ['JaxIndexError', 'JAXTypeError']))
+     if errorclass.endswith('Error') and errorclass not in ['JaxIndexError', 'JAXTypeError']
```

Comment:
Just out of curiosity, why not use `errorclass=[errorclass for errorclass in ...]` rather than the dict form?